plugins/UsersManager/lang/en.json

```diff
@@ -158,37 +158,37 @@
+        "AnonymousUserRoleChangeWarning": "Giving the %1$s user the %2$s role makes this website's data publicly available to everyone, even without a Matomo login.",
+        "GiveAccessToAll": "Give this user access to all websites",
         "OrManageIndividually": "Or manage this user's access to each website individually",
-        "ChangePermToAllSitesConfirm": "Are you sure you want to give the %1$s user %2$s access to every website you currently have admin access to?",
-        "ChangePermToAllSitesConfirm2": "Note: this will only affect the currently existing websites. New websites you may create will not be automatically accessible to this user.",
+        "ChangePermToAllSitesConfirm": "Give the %1$s user %2$s access to every website you currently have admin access to?",
+        "ChangePermToAllSitesConfirm2": "Note: This will only affect the currently existing websites. New websites you may create will not be automatically accessible to this user.",
         "CapabilitiesHelp": "Capabilities are individual abilities that users can be granted. Roles may, by default, grant certain capabilities. For example, the Admin role will automatically allow users to edit tags in the Tag Manager. For less powerful users, however, you can explicitly give users capabilities.",
         "Capabilities": "Capabilities",
-        "AreYouSureAddCapability": "Are you sure you want to give %1$s the %2$s capability for %3$s?",
-        "AreYouSureRemoveCapability": "Are you sure you want to remove the %1$s capability from %2$s for %3$s?",
+        "AreYouSureAddCapability": "Give %1$s the %2$s capability for %3$s?",
+        "AreYouSureRemoveCapability": "Remove the %1$s capability from %2$s for %3$s?",
         "IncludedInUsersRole": "Included in this user's role.",
         "Capability": "Capability",
-        "EmailChangeNotificationSubject": "Your Matomo account's email address has just been changed",
-        "EmailChangedEmail1": "The email address associated with your account has been changed to %1$s",
+        "EmailChangeNotificationSubject": "Your Matomo account's e-mail address has just been changed",
+        "EmailChangedEmail1": "The e-mail address associated with your account has been changed to %1$s",
         "EmailChangedEmail2": "This change was initiated from the following device: %1$s (IP address = %2$s).",
-        "IfThisWasYouIgnoreIfNot": "If this was you, feel free to ignore this email. If this was not you, please login, correct your email address, change your password and contact your Matomo administrator.",
-        "IfThisWasYouPasswordChange": "If this was you, feel free to ignore this email. If this was not you, please contact your Matomo administrator immediately, as your account might have been compromised!",
+        "IfThisWasYouIgnoreIfNot": "If this was not you, please log in, correct your e-mail address, change your password and contact your Matomo administrator.",
```

Comment:
Any reason for removing the first part?

Reply:
It is implied by what follows it. Also the neutral case is to do nothing, so no instruction is needed.

Reply:
hm. makes sense